gateway/ln-gateway/src/lib.rs

```diff
@@ -156,6 +156,17 @@ pub enum GatewayState {
     Disconnected,
 }
 
+impl From<GatewayState> for String {
+    fn from(gw_state: GatewayState) -> Self {
+        match gw_state {
+            GatewayState::Initializing => "Initializing".to_string(),
+            GatewayState::Configuring => "Configuring".to_string(),
+            GatewayState::Running { .. } => "Running".to_string(),
```

Comment:
nit: Maybe `GatewayState::LnConnected` and `GatewayState::LnDisconnected` instead of `GatewayState::Running` and `GatewayState::Disconnected`

Reply:
See https://github.com/fedimint/fedimint/pull/3135

I think the state that @bradleystachurski is adding is more appropriate to be called `LightningConnected` rather than this state

Reply:
makes sense